west.yml

```diff
@@ -102,7 +102,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 68bbdbdec1153355b14386d8f095058a496f66f4
+      revision: pull/58/head
```

Comment:
This PR depends on https://github.com/zephyrproject-rtos/hal_nxp/pull/58 being merged.

Reply:
Done.